modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -95,6 +95,14 @@
         </classes>
     </test>
 
+    <test name="apim-jwt-request-count-throttling-tests" preserve-order="true" parallel="false"
+          group-by-instances="true">
+        <classes>
+            <class name="org.wso2.am.integration.tests.throttling.JWTRequestCountThrottlingTestSuite"/>
```

Comment:
can we bring this into same JWT test suite

Reply:
Fixed in https://github.com/wso2/product-apim/pull/9484/commits/f5dffd1a8105d8ed30db34ae8cd35af8313b15e6